.github/dependabot.yml

```diff
@@ -21,48 +21,56 @@ updates:
   - package-ecosystem: "gomod"
     directory: "/"
     schedule:
-      interval: "weekly"
+      interval: "daily"
     commit-message:
       prefix: ":seedling:"
     labels:
       - "ok-to-test"
 
-  # Maintain dependencies for dockerfile in the branches
-  - package-ecosystem: docker
-    directory: "/build/thirdparty/darwin"
-    target-branch: "tools-releases"
+  # Maintain dependencies for go
+  - package-ecosystem: "gomod"
+    directory: "/testdata/project-v4"
     schedule:
-      interval: daily
+      interval: "daily"
     commit-message:
       prefix: ":seedling:"
-  - package-ecosystem: docker
-    directory: "/build/thirdparty/linux"
-    target-branch: "tools-releases"
+    labels:
+      - "ok-to-test"
+
+  # Maintain dependencies for go
+  - package-ecosystem: "gomod"
+    directory: "/testdata/project-v4-with-plugins"
     schedule:
-      interval: "weekly"
+      interval: "daily"
     commit-message:
       prefix: ":seedling:"
+    labels:
+      - "ok-to-test"
 
-  # Maintain dependencies for dockerfile scaffold in the projects
-  - package-ecosystem: docker
-    directory: "testdata/project-v3"
+  # Maintain dependencies for go
+  - package-ecosystem: "gomod"
+    directory: "/testdata/project-v4-multigroup"
     schedule:
-      interval: daily
+      interval: "daily"
     commit-message:
       prefix: ":seedling:"
+    labels:
+      - "ok-to-test"
+
+  # Maintain dependencies for dockerfile scaffold in the projects
   - package-ecosystem: docker
```

Comment:
this is the only entry for docker. Should every folder (`/`, `/testdata/project-v4`, `/testdata/project-v4-with-plugins`, `/testdata/project-v4-multigroup` and `docs/book/src/simple-external-plugin-tutorial/testdata/sampleexternalplugin/v1`) have dependabot checks for dockerfiles?

Reply:
Good point 👍

Reply:
Hi @mateusoliveira43 

The docs/book/src/simple-external-plugin-tutorial/testdata/sampleexternalplugin/v1 has no docker image
And the goal is be informed about some required change
The docker file is the sample for all samples so by just checking project-v4 we will be able to be notified if any change in the Dockerfile scaffold by Kubebuilder be required.

Reply:
understood